CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
For non-main branches, it is easier not to have a changelog entry for each PR.
This simplifies the rebasing process. Could you remove this, please?
I'll compose the full release notes for the next alpha in a single commit.